### ISSUE TITLE
Refactor ServicesController

### ISSUE THREAD

**Issue by Jelle-S** [StadGent/laravel_site_opening-hours#90]
**ServicesController::index()**
This method retrieves a list of all services the user has access to (both inactive and active). For the admin user that means **all** services that are in the database. This is **way** too much data to load at once. Laravel uses lazy loading, meaning that the data itself will only be loaded when the developer tries to access it. This means that for services the channels and opening hours will only be loaded when the developer (or the code) tries to access it. For the page that is being built here (the homepage when the use is logged in), all channels, openinghours **and** linked users are loaded (see https://github.com/StadGent/laravel_site_opening-hours/blob/develop/app/Repositories/ServicesRepository.php#L22 and https://github.com/StadGent/laravel_site_opening-hours/blob/develop/app/Repositories/ServicesRepository.php#L61 ). I aknowledge that this is needed to create the 'status' column on this page. Maybe we should think about what the added value of this field on this page is.
Also, this wouldn't be all that bad, if only the data required to display the page was loaded.
The pager and switching between active and inactive services is done in JavaScript. We could turn the pager in to an AJAX pager as well as make an AJAX call to switch between active and inactive services. Once the AJAX call is made, we can store that data on the client side so we don't have to make the call over and over again, and this will speed up the initial page load.

TLDR;
This method loads too much data and won't scale.
